### ISSUE TITLE
Responsive layout

### ISSUE THREAD

**Issue by fnadde42** [pixeline/bugs#21]
Bugs is a modern, simple and elegant issue tracker suitable not only for software engineers, but more importantly, by almost anyone who wants to get stuff done. Therefore, it has to cope with "everyone" technology such as smart phones and small displays.

Bugs has to be responsive.

**Comment by pixeline** [comment]
Easier said than done. Feel free to submit a patch that is responsive. Open source means collaboration, not commanding people to work free for you.

**Comment by fnadde42** [comment]
@pixeline I have been coding a bit so I know it's hard and not something that's prioritized. Just thought it might get to be on some low priority ToDo-list. And who knows, when I have learned PHP, Javascript and Git maybe I can submit a patch. Sure would love to but I don't want to push crappy code. I want to stress that I'm throwing out ideas on a open forum and not commanding anyone. 

Sorry if I stepped on anyone’s toes :S
I will try to learn and submit something useful.
